great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -1393,18 +1393,14 @@ def get_connection(self) -> sqlalchemy.Connection:
         Returns:
             Sqlalchemy connection
         """
-        if self.dialect_name in _PERSISTED_CONNECTION_DIALECTS:
-            try:
-                if not self._connection:
-                    self._connection = self.engine.connect()
-                yield self._connection
-            finally:
-                # Temp tables only persist within a connection for some dialects,
-                # so we need to keep the connection alive.
-                pass
-        else:
-            with self.engine.connect() as connection:
-                yield connection
```

Comment:
Point of potential discussion : passing around a single connection by default? (instead of only doing it for `mssql` and `sqlite`